stackslib/src/net/api/poststackerdbchunk.rs

```diff
@@ -145,6 +145,14 @@ impl StackerDBErrorCodes {
             "reason": self.reason()
         })
     }
+
+    pub fn from_code(code: u32) -> Option<Self> {
+        match code {
+            0 => Some(Self::DataAlreadyExists),
+            1 => Some(Self::NoSuchSlot),
+            _ => None,
+        }
+    }
```

Comment:
@netrome this is what's needed to fix the output mutants check. You can find this by viewing [the "Summary" page on that check](https://github.com/stacks-network/stacks-core/actions/runs/8177496330?pr=4348)

```suggestion
    #[cfg_attr(test, mutants::skip)]
    pub fn from_code(code: u32) -> Option<Self> {
        match code {
            0 => Some(Self::DataAlreadyExists),
            1 => Some(Self::NoSuchSlot),
            _ => None,
        }
    }
```

Reply:
Oh, thanks. I missed that. :pray: